src/assets/stylesheets/_social_cards.scss

```diff
@@ -83,6 +83,97 @@
   width: auto;
 }
 
+// -- validations -- //
+
+$red: #c04e4e;
+$yellow: #ff9f00;
+$yellowDarken: #cc7f00;
+$blue: #00addf;
+$blueDarken: #0085ac;
+
+.scrivito_detail_label {
+  &.scrivito_error {
+    span,
+    .scrivito_icon,
+    .scrivito_icon:hover {
+      color: $red;
+    }
+  }
+  &.scrivito_warning {
+    span,
+    .scrivito_icon,
+    .scrivito_icon:hover {
+      color: $yellowDarken;
+    }
+  }
+  &.scrivito_info {
+    span,
+    .scrivito_icon,
+    .scrivito_icon:hover {
+      color: $blue;
+    }
+  }
+}
+
+.scrivito_validation_notice,
+.seo_card_input .input {
+  line-height: 27px;
+  margin-bottom: 5px;
+  box-shadow: 0 0 3px rgba(0, 0, 0, 0.2);
+  border-radius: 2px;
+  background: rgba(255, 255, 255, 0.6);
```

Comment:
Where are this styles coming from?

Reply:
Values almost all come from the SDK (corrected a padding and line height), just adjusted some classes/selectors to the different structure of the social cards tab.